labelbox/schema/project.py

```diff
@@ -1127,6 +1108,99 @@ def batches(self) -> PaginatedCollection:
             cursor_path=['project', 'batches', 'pageInfo', 'endCursor'],
             experimental=True)
 
+    def task_queues(self) -> List[TaskQueue]:
+        """ Fetch all task queues that belong to this project
+
+        Returns:
+            A `List of `TaskQueue`s
+        """
+        query_str = """query GetProjectTaskQueuesPyApi($projectId: ID!) {
+              project(where: {id: $projectId}) {
+                taskQueues {
+                  %s
+                }
+              }
+            }
+        """ % (query.results_query_part(Entity.TaskQueue))
+
+        task_queue_values = self.client.execute(
+            query_str, {"projectId": self.uid},
+            timeout=180.0,
+            experimental=True)["project"]["taskQueues"]
+
+        return [
+            Entity.TaskQueue(self.client, field_values)
+            for field_values in task_queue_values
+        ]
+
+    def move_data_rows_to_task(self, data_row_ids: List[str],
```

Comment:
nit: `move_data_rows_to_task_queue` sounds more accurate

Reply:
I'll update it to `move_data_rows_to_task_queue`, but keep the action as `move` for consistency with the UI equivalent